v3/fleet-local/control/chronos@.service

```diff
@@ -29,6 +29,7 @@ ExecStartPre=-/usr/bin/docker rm chronos
 ExecStart=/usr/bin/sh -c "docker run \
   --name chronos \
   --net=host \
+  --read-only \
```

Comment:
Can you link to the reference for this option? I cannot seem to find it anywhere